R/utils.R

```diff
@@ -142,3 +142,23 @@ is_windows <- function() {
 }
 
 compact <- function(x) x[!vapply(x, is.null, logical(1))]
+
+set_odbcsysini <- function() {
+  odbcsysini <- Sys.getenv("ODBCSYSINI")
+
+  if (!identical(odbcsysini, "")) {
+    return(invisible(TRUE))
+  }
```

Comment:
```suggestion
  odbcsysini <- Sys.getenv("ODBCSYSINI")
  if (!identical(odbcsysini, "")) {
    return(invisible(TRUE))
  }
```

Just because it makes it look more like a "paragraph" of related code